src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -85,8 +85,124 @@ def do_observation(self, event: Event) -> None:
         """Define the observations in the concrete class"""
         pass
 
+    # TODO: consider using compressed csv instead of hdf
     def on_simulation_end(self, event: Event) -> None:
-        self.responses.to_hdf(self.output_dir / self.output_filename, key="responses")
+        self.responses.to_hdf(self.output_dir / self.output_filename, key="data")
+
+
+class HouseholdSurveyObserver(BaseObserver):
+
+    # FIXME: Do we want these household survey constants here or in constants.metadata?
+    INPUT_COLS = [
+        "alive",
+        "household_id",
+        "housing_type",
+        "first_name",
+        "middle_name",
+        "last_name",
+        "age",
+        "sex",
+        "race_ethnicity",  # For simulant omission
+        "date_of_birth",
+        "address_id",
+        "state",
+        "puma",
+        "guardian_1",  # For noise functions
+        "guardian_2",  # For noise functions
+    ]
+    OUTPUT_COLS = [
+        "survey",
+        "survey_date",
+        "household_id",
+        "housing_type",
+        "first_name",
+        "middle_initial",
+        "last_name",
+        "age",
+        "sex",
+        "race_ethnicity",  # For simulant omission
+        "date_of_birth",
+        "address_id",
+        "state",
+        "puma",
+        "guardian_1",  # For noise functions
+        "guardian_2",  # For noise functions
+        "guardian_1_address_id",  # For noise functions
+        "guardian_2_address_id",  # For noise functions
+    ]
+    SAMPLING_RATE = {
+        "ACS": 12,  # FIXME: 12000
+        "CPS": 60,  # FIXME: 60000
+    }
```

Comment:
We have a lot of components that require different constant list, maps, etc and we also talked about having a list for all the state table columns.  I think we should think about adding something to data values and/or metadata similar to ata keys but for constants to help organize this.  Obviously not for this PR.

Reply:
Well this is unique b/c it's very specific to this particular observer and these values will likely not be used outside of this spot. If a value is used in more than one place, then I def agree there should be a centralized location of constants (column names are the big obvious here). It's less useful and perhaps even worse if you only use it once.